enumerable.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Enumerable
+  # parent block method is the #each
+  # num_arr = Array(1..9)
+  # num_arr.each { |x| puts x.to_s }
```

Comment:
Let's remove commented out code, to check old versions of your code, use git history.